contrib/hack/makeKubectl.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
In general this seems like content we could push down into repo-infra.  I don't want to do that here, but let's extract the portable bits (if you agree that this is really more a function of the fact that we need a kubectl than anything specific to service-catalog) and put them into repo-infra.